rbac/processor/event_handler.py

```diff
@@ -119,8 +125,19 @@ def namespaces(self):
         return addresser.family.namespaces
 
     def apply(self, transaction, state):
-        payload = RBACPayload()
-        payload.ParseFromString(transaction.payload)
+        try:
+            payload = RBACPayload()
+            payload.ParseFromString(transaction.payload)
+
+            if self._processor.has_message_handler(message_type=payload.message_type):
+                return self._processor.handle_message(
+                    header=transaction.header, payload=payload, state=state
+                )
+        except ValueError as err:
+            raise InvalidTransaction(err)
+        except Exception as err:  # pylint: disable=broad-except
```

Comment:
Does this block require catching the generic Exception type? If not, we can allow it to break naturally on an unexpected error if/when that time comes and respond with specific exception handling.

Reply:
broad exception is required, transaction processor can't fail because of a message failure